rmgpy/species.py

```diff
@@ -839,6 +847,40 @@ def set_structure(self, structure):
                     raise
             self.generate_resonance_structures()
 
+    def get_henry_law_constant_data(self):
+
+        if self.henry_law_constant_data:
+            return self.henry_law_constant_data
+
+        if diffusion_limiter.enabled:
+
+            self.henry_law_constant_data = diffusion_limiter.get_henry_law_constant_data(self)
```

Comment:
Why is this attached to the diffusion_limiter?

Reply:
kH requires the solvent to be specified, which I think means the diffusion limiter is enabled, so I thought it would make sense to tie them together. Where do you think is a better place to put this function?

Reply:
I'm not sure, but I certainly wouldn't ever expect liq-vap stuff to be attached to the diffusion limiter structurally. I'm not sure I understand what problem you're getting around structurally by doing this, but it might be better as its own thing or to rename the diffusion limiter to something more general that covers both.